src/CIG-Tests/CigPharoFunctionTemplateTest.class.st

```diff
@@ -74,7 +74,7 @@ CigPharoFunctionTemplateTest >> testGenerateFunctionWithReturnString [
 		assert: template generate
 		equals: 'fadeColor: color alpha: alpha
 
-	self ffiCall: #(Color Fade(Color color, float alpha))'
+	^ self ffiCall: #(Color Fade(Color color, float alpha))'
```

Comment:
This represents my case: The method was generated without the ^ and I needed it, to obtain the result.